test/gerkin/features/db_delete.feature

```diff
@@ -1,22 +1,14 @@
 Feature: API endpoint allowing users to delete Digital registries schema.
   Request endpoint: DELETE /database/{id}/
 
-  Background:
-    Given User wants to delete the Digital Registries schema
-
   Scenario: The user successfully deletes the Digital Registries schema
-    Given The requested database schema exists
-    When The user triggers an action to delete the database schema
-    And The request with a valid payload is sent
-    Then Operation finishes successfully
-
-  Scenario: The user is not able to delete the Digital Registries schema
-    Given The requested database schema does not exists
-    When The user triggers an action to delete the database schema
-    And The request with a valid payload is sent
-    Then Operation finishes successfully
-
-  Scenario: The user is not able to delete Digital Registries schema
-    When The user triggers an action to delete the database schema
-    And The request with an invalid payload is sent
-    Then Operation results in an error'
+    Given User wants to delete the Digital Registries schema with id=12345
+    And The requested database schema exists
+    When The user triggers an action to delete the database schema with id=12345
+    Then Operation to a delete database schema finishes successfully
+
+  Scenario: The user is not able to delete the Digital Registries schema because schema does not exist
+    Given User wants to delete the Digital Registries schema with id=12
+    And The requested database schema does not exists
+    When The user triggers an action to delete the database schema with id=12
+    Then Operation to delete a database schema finishes successfully
```

Comment:
@ingmarvali should a failed delete really result in a "success" (and what does that mean in this context)?

Indeed our OpenAPI specs don't define error responses yet either (https://github.com/ingmarvali/BuildingBlockAPI/blob/main/DigitalRegistriesBB/GovStack_Digital_registries_BB_Database_API_template-1.1.0.json as linked from GitBook BB Spec)